CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 1.7.2
+
+NEW FEATURES:
+
+ENHANCEMENTS:
+
+BUG FIXES:
+* Fix e2e test_test on darwin amd64 ([#1551](https://github.com/opentofu/opentofu/pull/1551))
```

Comment:
Do we think this type of change should be included in the changelog?

Reply:
We don't usually

Reply:
removed it